docs/getting-started/1242-install.mdx

```diff
@@ -156,6 +157,41 @@ If you want to dig into the details of how Dagger works, we recommend you read o
 
 <Button label="How It Works" docPath="f44rm/how-it-works" />
 
+export const Templates = ({ templates }) => {
```

Comment:
This is actually made in purpose so the team has a sort of interface where they can modify content easily. Instead of modifying the component, they can modify the data object at the bottom, making it easier to implement changes without our help. For example, the content in the Main Cards in `overview.mdx` were already easily changed by the team because of this.

Reply:
I understand. So what we should do is comment our code instead:
```
/// Templates ////
....
/// TemplateCard ///
...
etc..
```
and the day we need those components somewhere else, let's export them. By the way, why are they exported ?

Reply:
> I understand. So what we should do is comment our code instead. Let's not make the code more complexe than it should be.
> 
> ```
> /// Templates ////
> ....
> /// TemplateCard ///
> ...
> etc..
> ```
> 
> and the day we need those components somewhere else, let's export them. By the way, why are they exported ?

I'm not sure I follow. If we comment the Templates and TemplateCard components, they would disappear from the Install page.
They are exported because otherwise [Docusaurus would throw an error](https://docusaurus.io/docs/next/markdown-features/react#exporting-components).

Reply:
Oh, yes. Now I see it, you are totally right. I'll merge both components into one.